scss/tech.scss

```diff
@@ -0,0 +1,59 @@
+@import "colors";
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+.boxes {
+  display: flex;
+  flex-flow: row;
+  flex-wrap: wrap;
+  padding: 0px 0px 20px 100px;
+  margin-bottom: 50px;
+}
+
+.box {
+  background: $green;
+  border-radius: 8px;
+  height: 310px;
+  margin: 25px;
+  position: relative;
+  text-align: center;
+  width: 200px;
+}
+
+.box::before {
+  border: 1px solid $green;
+  border-radius: 8px;
+  right: 10px;
+  bottom: 10px;
+  content: '';
+  height: 310px;
+  position: absolute;
+  width: 200px;
+  z-index: -1;
+}
+
+.box-title {
+  font-size: $text-medium;
+  margin: 10px;
+}
+
+.box-release {
+  font-size: $text-small;
+  font-weight: bold;
+  margin:10px;
+}
+
+.box-description {
+  font-size: $text-small;
+  margin:22px;
+}
+
+.box-cover {
+  height: $tech-logo-size;
+  margin:5px;
+  width: $tech-logo-size;
```

Comment:
by setting both width and height images might be deformed. use the `object-fit` property to fix this.